backend/tests/unit/memberstack.test.js

```
@@ -427,12 +469,20 @@ describe('/v1/auth/refresh-token endpoint', () => {
     );
     if (!layer) {
       // Add logging for debugging if the correct layer is not found
-      console.error('Could not find POST /refresh-token route layer in authRouter.stack');
-      console.error('authRouter.stack:', authRouter.stack.map(l => l.route && l.route.path));
-      throw new Error('Test setup error: Could not find POST /refresh-token route layer');
+      console.error(
+        'Could not find POST /refresh-token route layer in authRouter.stack'
+      );
+      console.error(
+        'authRouter.stack:',
+        authRouter.stack.map(l => l.route && l.route.path)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)